src/concrete/ml/deployment/fhe_client_server.py

```diff
@@ -251,11 +251,14 @@ def load(self):  # pylint: disable=no-value-for-parameter
                 versions = json.load(file)
 
         errors = []
-        packages_to_check = {"concrete-python"}
+        packages_to_check = {"concrete-python", "concrete-ml"}
```

Comment:
actually could this section be refactored into a function ? I see that you seem to use the same lines for both the server and client classes

Reply:
It could indeed, in another PR is that fine with you.

They are already quite a bit of changes here.

Reply:
yes sure : https://github.com/zama-ai/concrete-ml-internal/issues/3993